web/src/components/modals/AutomaticUpdatesModal.jsx

```diff
@@ -97,10 +101,10 @@ export default class AutomaticUpdatesModal extends React.Component {
       selectedSchedule,
       selectedAutoDeploy,
     };
+  }
 
-    if (this.state.updateCheckerSpec === "") {
```

Comment:
why is this not needed anymore? is it never empty?

Reply:
yeah, there is a check above to find "custom" or "default" now

Reply:
but that's for the selected schedule, not the actual cron spec being used, right?

Reply:
the actual cron spec should also displayed as `@default` i believe when it's empty. the "custom" or "default" are for the drop down options